### DISCUSSION TITLE
Automated Declining for Changes-Requested?

### DISCUSSION THREAD

**Discussion by DrKJeff16** [tosdr/edit.tosdr.org#980] [Ideas] [answered]
# Why The Suggestion

As established, we have approved a [new policy](https://forum.tosdr.org/t/new-changes-requested-policy/221) for Phoenix, in which points that get marked with `Changes-Requested` will be deleted if they haven't been updated two months after said status is set.

# Is It Possible To Make It Automated?

Is it feasible to create a bot-related process that automatically declines points after the passing of 60 days? Would make things a lot easier for Reviewers and Curators, specially if also done retroactively...

**Comment by JustinBack** [comment]
This should be easily integrated with rake tasks and schedule them on heroku

**Comment by madoleary** [comment]
Hey guys, yes, this is totally feasible. I can try to take a look this week.

**Comment by JustinBack** [comment, marked as answer]
This has been implemented now!
https://github.com/tosdr/edit.tosdr.org/blob/c117eff20d1f301bc923c6a3b8956499c876cfaa/lib/tasks/points.rake#L1-L34
